main.tf

```diff
@@ -100,3 +100,35 @@ resource "snowflake_account_grant" "this" {
 
   with_grant_option = false
 }
+
+resource "snowflake_grant_privileges_to_role" "this" {
```

Comment:
looking at the `each` directive, this should be named not `this` but `dynamic_tables` something?